### PR TITLE
fix: tooltip, text, table

### PR DESCRIPTION
**Pull request by yqrashawn**
1. fix tooltip posotion issue
2. fix text and table classname issue

---

## Checklist

- [x] Fix linting errors
- [x] Tests have been added / updated (or snapshots)

### CODE REVIEW

File: components/text/text.tsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ const Text: React.FC<React.PropsWithChildren<TextProps>> = ({
   }, [renderableChildElements, children, size])
 
   return (
-    <TextChild className={className} tag={tag} size={size} {...props}>
+    <TextChild className={`text ${className}`} tag={tag} size={size} {...props}>
```
Comment (AngeliaQ) [inline components/text/text.tsx:120]: TextProps should not export defaultProps

**Review by AngeliaQ — CHANGES_REQUESTED** [review]

File: components/text/text.tsx (hunk 1 above)
Comment (yqrashawn) [reply components/text/text.tsx:120]: done

File: components/table/table.tsx [hunk 2]
```diff
@@ -364,7 +364,9 @@ function Table<RecordType>({ variant, children, ...props }: TableProps<RecordTyp
     <div>
       <RcTable<RecordType>
         prefixCls="table"
-        className={`${className} ${variant === 'solid' ? 'variant-solid' : 'variant-line'}`}
+        className={`${className} ${innerStyleClassName} ${
```
Comment (angelia-yuqi-personal) [inline components/table/table.tsx:367]: classname should be placed after innerStyleClassName

**Review by AngeliaQ — APPROVED** [review]

**Review by 0x74616e67 — APPROVED** [review]

**Comment by AngeliaQ** [comment]
Task linked: [CU-4ntwf6 tooltip 定位有问题](https://app.clickup.com/t/4ntwf6)

**Pull request merged**